### PR TITLE
Add documentation for the simple application mode

### PR DESCRIPTION
**Pull request by sbrunner**
GSGMF-1535

### CODE REVIEW

File: doc/integrator/admin_interface.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Configure the admin interface
 =============================
 
+.. note::
+
+    This is not possible in the simple application mode
```
Comment (jwkaltz) [inline doc/integrator/admin_interface.rst:8]: Suggestion, be more explicit:
"Configuration of the admin interface is not possible in the simple application mode."

File: doc/integrator/create_application.rst.tmpl [hunk 2]
```diff
@@ -120,12 +120,14 @@ Simple application
 
 A simple application is an application for which no custom code is needed (Python or JavaScript).
 
+See also :ref:`integration_structure` for more informations.
```
Comment (jwkaltz) [inline doc/integrator/create_application.rst.tmpl:123]: informations -> information

File: doc/integrator/create_application.rst.tmpl [hunk 3]
```diff
@@ -120,12 +120,14 @@ Simple application
...
+
+All the administrator in integrator guides indicate if it doesn't consern the simple application mode.
```
Comment (jwkaltz) [inline doc/integrator/create_application.rst.tmpl:125]: I don't understand this sentence

File: doc/integrator/extend_data_model.rst [hunk 4]
```diff
@@ -1,6 +1,10 @@
 Extend the data model
 ---------------------
 
+.. note::
+
+    This is not possible in the simple application mode
```
Comment (jwkaltz) [inline doc/integrator/extend_data_model.rst:6]: Same as above, let's be more explicit.
"Extending the data model is not possible in the simple application mode."

File: doc/integrator/internationalization.rst [hunk 5]
```diff
@@ -5,7 +5,8 @@
 Internationalization
 ====================
 
-In the file ``<package>.mk``, define the supported languages with (default):
+In the file ``<package>.mk``, define the supported languages with (default),
+in simple mode it's fixed to (en fr de it):
```
Comment (jwkaltz) [inline doc/integrator/internationalization.rst:9]: "In the simple application mode, the list of languages is constant: en fr de it"

File: doc/integrator/ngeo.rst.tmpl [hunk 6]
```diff
@@ -11,16 +11,30 @@ The main page where we can redefine the header is in the file:
 ``geoportal/<package>_geoportal/static-ngeo/js/apps/Controller<interface>.js``
 where ``<interface>`` is the interface name.
 
-The style sheet file is ``geoportal/<package>_geoportal/static-ngeo/js/apps/sass/<interface>.scss``.
+In all mode:
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:14]: What is the all mode?

File: doc/integrator/ngeo.rst.tmpl [hunk 7]
```diff
@@ -175,6 +197,10 @@ for the HTML.
 Extending or overriding an existing component
 ----------------------------------------------
 
+.. note::
```
Comment (jwkaltz) [inline doc/integrator/ngeo.rst.tmpl:200]: There are so many points saying "not possible in the simple application mode" that I wonder if we should create a table instead:
containing list of things that can be done, and then a checkbox "available in simple application mode yes/no"

File: doc/integrator/structure.rst [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+image.
+
+``tilegeneration``: all the files needed by TileCloud-chain, will be in the folder ``/etc/tilegeneration``
+in the config image.
+
+``print/print-apps``: all the files needed my Mapfish print, will be in the folder
+``/usr/local/tomcat/webapps/ROOT/print-apps`` in the config image.
+
+``geoportal/vars*.yaml``, ``geoportal/CONST_vars.yaml``, ``geoportal/CONST_config-schema.yaml``: All the
+file needed to build the config, will be in the file ``/etc/geomapfish/config.yaml`` in the config image.
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:26]: file needed -> files needed

File: doc/integrator/structure.rst [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+``geoportal/geomapfish_geoportal/static``: All the static file needed for your application, will be
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:28]: file -> files

File: doc/integrator/structure.rst [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+available ins the static vies with cache buster. Will be in the folder ``/etc/geomapfish/static`` in the
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:29]: ins -> in

Comment (jwkaltz) [inline doc/integrator/structure.rst:29]: what is vies?
view?

File: doc/integrator/structure.rst [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+``geoportal/geomapfish_geoportal/locale``: Call the files needed to localise your application, will be in
+the folder ``/etc/geomapfish/locale`` in the config image.
+
+Custom application
+------------------
+
+.. note::
+
+    This part didn't exists in the simple application mode
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:40]: Suggestion:
This section is not relevant in the simple application mode.

File: doc/integrator/structure.rst [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+``geoportal/geomapfish_geoportal`` the pyramid application.
+
+``geoportal/webpack.*`` the webpack build configuration.
+
+other files in ``geoportal`` exxantially files rlated to the build.
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:46]: exxantially files rlated to the build.
probably:
"essentially files related to the build" ?

File: doc/integrator/structure.rst [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+In the image everithing will be in the ``/app`` folder exept of the result of the webpack build who will be
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:48]: everithing -> everything
exept of the -> except the
who will -> which will

File: doc/integrator/structure.rst [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+in the directory ``/etc/static-ngeo/``.
+
+Other files
+-----------
+
+The files in ``CONST_create_template`` is the clean instantiation or the create saffold durring the last
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:54]: saffold durring -> scaffold during

File: doc/integrator/structure.rst [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+upgrade. It is used the create the diff durring the migration.
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:55]: durring -> during

File: doc/integrator/structure.rst [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+
+The files in ``.github``, ``ci``, or dirrectily in the project base are used by the CI, by the IDE, or by
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:57]: dirrectily -> directly

File: doc/integrator/structure.rst [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+the build part out of the Docker.
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:58]: the Docker -> Docker

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+github/workflows/main.yaml_tmpl [hunk 18]
```diff
@@ -68,16 +68,22 @@ jobs:
 
       - name: Build geoportal
         run: ./build --geoportal
+        if: hashFiles('geoportal/Dockerfile') != ''
```
Comment (jwkaltz) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+github/workflows/main.yaml_tmpl:71]: Shouldn't these kind of changes be in a separate PR?

File: doc/integrator/ngeo.rst.tmpl (hunk 6 above)
Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl:14]: simple and normal or complex (it hasn't a defined name...)

File: doc/integrator/ngeo.rst.tmpl (hunk 7 above)
Comment (sbrunner) [reply doc/integrator/ngeo.rst.tmpl:200]: I think no, for me, we should see that when we read the concerned documentation...

File: doc/integrator/structure.rst (hunk 10 above)
Comment (sbrunner) [reply doc/integrator/structure.rst:29]: Yes :-)

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+github/workflows/main.yaml_tmpl (hunk 18 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+github/workflows/main.yaml_tmpl:71]: Yes I can :-)

File: doc/integrator/create_application.rst.tmpl (hunk 3 above)
Comment (sbrunner) [reply doc/integrator/create_application.rst.tmpl]: Better?:
All the administrator and integrator guides indicate if the section doesn't consern the simple
application mode.

File: doc/integrator/ngeo.rst.tmpl (hunk 6 above)
Comment (jwkaltz) [reply doc/integrator/ngeo.rst.tmpl:14]: how about "full application mode" (vs. "simple application mode")?

File: doc/integrator/create_application.rst.tmpl [hunk 19]
```diff
@@ -120,12 +120,15 @@ Simple application
 
 A simple application is an application for which no custom code is needed (Python or JavaScript).
 
+See also :ref:`integrator_structure` for more information.
+
+All the administrator and integrator guides indicate if the section doesn't concern the simple
```
Comment (jwkaltz) [inline doc/integrator/create_application.rst.tmpl:125]: Suggestion:
"The documentation sections indicate whether the current section applies to the simple application mode or not."

File: doc/integrator/create_application.rst.tmpl [hunk 20]
```diff
@@ -120,12 +120,15 @@ Simple application
 
 A simple application is an application for which no custom code is needed (Python or JavaScript).
```
Comment (jwkaltz) [inline doc/integrator/create_application.rst.tmpl:121]: Suggestion:
add: "In contrast, a _full application_ is an application for which the all possibilities for customization are made available".

File: doc/integrator/ngeo.rst.tmpl (hunk 6 above)
Comment (jwkaltz) [reply doc/integrator/ngeo.rst.tmpl:14]: Suggestion:
"In all application modes (simple and full):"

File: doc/integrator/structure.rst [hunk 21]
```diff
@@ -0,0 +1,58 @@
...
+
+Project structure
+=================
+
+We can split the project in 3 parts.
+
+Configuration
+-------------
+
+Thoes files are use to build the config image.
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:11]: Thoes -> These

File: doc/integrator/structure.rst [hunk 22]
```diff
@@ -0,0 +1,58 @@
...
+
+``mapserver``: all the files needed by Mapserver, will be in the folder ``/etc/mapserver`` in the config
+image.
+
+``qgisserver``: all the files needed by QGIS server, will be in the folder ``/etc/qgisserver`` in the config
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:16]: remove the comma

File: doc/integrator/structure.rst [hunk 23]
```diff
@@ -0,0 +1,58 @@
...
+image.
+
+``tilegeneration``: all the files needed by TileCloud-chain, will be in the folder ``/etc/tilegeneration``
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:19]: remove the comma

File: doc/integrator/structure.rst [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+in the config image.
+
+``print/print-apps``: all the files needed my Mapfish print, will be in the folder
+``/usr/local/tomcat/webapps/ROOT/print-apps`` in the config image.
+
+``geoportal/vars*.yaml``, ``geoportal/CONST_vars.yaml``, ``geoportal/CONST_config-schema.yaml``: All the
+files needed to build the config, will be in the file ``/etc/geomapfish/config.yaml`` in the config image.
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:26]: remove the comma

File: doc/integrator/structure.rst [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+
+``geoportal/geomapfish_geoportal/static``: All the static files needed for your application, will be
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:28]: remove the comma

File: doc/integrator/structure.rst [hunk 26]
```diff
@@ -0,0 +1,58 @@
...
+available in the static view with cache buster. Will be in the folder ``/etc/geomapfish/static`` in the
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:29]: Something wrong with the sentence (. Will be in the folder...)

File: doc/integrator/structure.rst [hunk 27]
```diff
@@ -0,0 +1,58 @@
...
+config image.
+
+``geoportal/geomapfish_geoportal/locale``: Call the files needed to localise your application, will be in
```
Comment (jwkaltz) [inline doc/integrator/structure.rst:32]: What is meant with "Call"?
Contains?

**Review by jwkaltz — APPROVED** [review]
Some suggestions for improvements

**Comment by sbrunner** [comment]
Tanks @jwkaltz for the review :-)
